specification/mediaservices/resource-manager/Microsoft.Media/stable/2021-06-01/AccountFilters.json

```diff
@@ -162,6 +162,7 @@
           "items": {
             "$ref": "#/definitions/FilterTrackPropertyCondition"
           },
+          "x-ms-identifiers": [],
           "description": "The track selections."
```

Comment:
There is one more array in this file, AccountFilterCollection->value. does this not need x-ms-identifiers?